thriftpy2/contrib/aio/rpc.py

```diff
@@ -30,22 +31,34 @@ async def make_client(service, host='localhost', port=9090, unix_socket=None,
         host = parsed_url.hostname or host
         port = parsed_url.port or port
     if unix_socket:
-        socket = TAsyncSocket(unix_socket=unix_socket,
-                              connect_timeout=connect_timeout,
-                              socket_timeout=timeout)
+        socket_family = socket.AF_UNIX if not socket_family else socket_family
+        client_socket = TAsyncSocket(
+            unix_socket=unix_socket,
+            connect_timeout=connect_timeout,
+            socket_timeout=timeout,
+            socket_family=socket_family,
+        )
```

Comment:
Ok, I think I should not pass the socket_family option when using unix_socket.

```suggestion
        client_socket = TAsyncSocket(
            unix_socket=unix_socket,
            connect_timeout=connect_timeout,
            socket_timeout=timeout
        )
```

Reply:
I will rewrite this part later today.

Reply:
Exactly